test/testrunner.py

```diff
@@ -82,16 +204,21 @@ def setup_eks_cluster(framework_name):
 def setup_sm_benchmark_env(dlc_images, test_path):
     # The plan is to have a separate if/elif-condition for each type of image
     if "tensorflow-training" in dlc_images:
-        tf1_images_in_list = (re.search(r"tensorflow-training:(^ )*1(\.\d+){2}", dlc_images) is not None)
-        tf2_images_in_list = (re.search(r"tensorflow-training:(^ )*2(\.\d+){2}", dlc_images) is not None)
+        tf1_images_in_list = re.search(r"tensorflow-training:(^ )*1(\.\d+){2}", dlc_images) is not None
+        tf2_images_in_list = re.search(r"tensorflow-training:(^ )*2(\.\d+){2}", dlc_images) is not None
         resources_location = os.path.join(test_path, "tensorflow", "training", "resources")
         setup_sm_benchmark_tf_train_env(resources_location, tf1_images_in_list, tf2_images_in_list)
 
 
 def main():
     # Define constants
+    start_time = datetime.now()
     test_type = os.getenv("TEST_TYPE")
-    dlc_images = get_dlc_images()
+    executor_mode = os.getenv("EXECUTOR_MODE", "False")
```

Comment:
Check if `os.getenv(name, "False").lower() == "true"` here, assign to `executor_mode`, and directly use `if executor_mode:` elsewhere.

Reply:
fixed